.vscode/tasks.json

```diff
@@ -0,0 +1,54 @@
+{
+	"version": "2.0.0",
+	"tasks": [
+		{
+			"type": "dreammaker",
+			"dme": "paradise.dme",
+			"problemMatcher": [
+				"$dreammaker"
```

Comment:
I wasnt able to get this file to work for me, but it should probably be fine, especially for new devs setting up their workspace.

Reply:
Strange, was it any of the build tasks or just the DM one?

Reply:
Ah, nevermind I got it working. Its CTRL+SHIFT+B and not an option in the launch configuration. Perhaps some documentation on that somewhere would be good